packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -330,4 +320,12 @@ export class RulesetExecutor {
       rulesResultsSubject$: result$
     } as EngineRuleset;
   }
+
+  /**
+   * Plug ruleset to fact streams and trigger a first evaluation
+   * @deprecated This function is not made to be accessible from Outside of the class, will be removed in v11
```

Comment:
I think we can remove it in v10

Reply:
v10 or v12 :) not v11 :D

Reply:
v10 really is too short and v12 is a bit too far (for a function that does not make sense to be used).
If the reason to not be v11 is that "we do not make change in v11", that does not make sens to flag it this realese to remove it in v10 neither...
I don't want to argue and change it to `v10` but we may think to be less rigid on that "impair" release.

Reply:
ok for me, I see your point